types/raf/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for raf 3.4
+// Project: https://github.com/chrisdickinson/raf#readme
+// Definitions by: Ben Lorantfy <https://github.com/BenLorantfy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const raf: {
+  (callback: (timestamp: number) => void): number;
+  cancel: (handle: number) => void;
+  polyfill: (globalObject?: any) => void;
+};
+
+export default raf;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
fixed